docs/en/connector-v2/sink/Maxcompute.md

```diff
@@ -0,0 +1,80 @@
+# Maxcompute
+
+> Maxcompute sink connector
+
+## Description
+
+Used to read data from Maxcompute.
+
+## Key features
+
+- [x] [batch](../../concept/connector-v2-features.md)
+- [ ] [exactly-once](../../concept/connector-v2-features.md)
+- [ ] [schema projection](../../concept/connector-v2-features.md)
+- [x] [parallelism](../../concept/connector-v2-features.md)
+
+## Options
+
+| name                    | type    | required  | default value |
+|-------------------------|---------|-----------|---------------|
+| accessId                | string  | yes       | -             |
+| accesskey               | string  | yes       | -             |
+| endpoint                | string  | yes       | -             |
+| project                 | string  | yes       | -             |
+| result_table_name       | string  | yes       | -             |
```

Comment:
`result_table_name` is in `common-options` now. 

Here is an example

<img width="775" alt="image" src="https://user-images.githubusercontent.com/32193458/205663258-728e5865-dd79-473e-a47e-6f5a28d7e56c.png">